app/views/users/two_factor_authentication_setup/index.html.erb

```diff
@@ -20,7 +20,7 @@
                     url: authentication_methods_setup_path do |f| %>
   <div class="margin-bottom-4">
     <fieldset class="margin-0 padding-0 border-0">
-      <legend class="margin-bottom-2 usa-sr-only"><%= @presenter.intro %></legend>
+      <legend class="margin-bottom-2 usa-sr-only"><%= @presenter.form_legend %></legend>
```

Comment:
Since it's static content not varying on any conditions, I might suggest just adding the translation here, to avoid the indirection of a method.

```suggestion
      <legend class="margin-bottom-2 usa-sr-only">
        <%= t('two_factor_authentication.form_legend') %>
      </legend>
```

Reply:
That sounds good. I made that revision. Thank you!